dev-docs/source/conf.py

```diff
@@ -42,11 +42,11 @@ def setup(app):
 # The version info for the project you're documenting, acts as replacement for
 # |version| and |release|, also used in various other places throughout the
 # built documents.
-
+version_str = mantid.__version__
 # The short X.Y version.
-version = "master"
+version = ".".join(version_str.split(".")[:2])
 # The full version, including alpha/beta/rc tags.
-release = version
+release = version_str
```

Comment:
Sorry to be picky but can we rename `version_str` to something like `full_version` and `version` to `short_version` or something like that? It's a bit unclear to have `version_str` and `version`.

Reply:
Yes no problem. I need to keep `version` the same because it is the name of a configuration variable for sphinx https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-version

I have renamed the full version to be `release`, which is in line with what the sphinx configuration wants